src/container/BooksList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+
+const BooksList = ({ bookId, title, category }) => (
+  <table>
+    <th>BookId</th>
+    <th>Title</th>
+    <th>Category</th>
+    <Book bookId={bookId} title={title} category={category} />
```

Comment:
- [ ] [REQUIRED] This has caused an JSX error syntax. Please work on it.
![1](https://user-images.githubusercontent.com/58552748/132347610-fbd1dc73-c307-4957-930f-64945e65ce07.jpg)